contracts/test/LibLiquidationMock.sol

```diff
@@ -72,4 +76,29 @@ library LibLiquidationMock {
             minimalReceipt
         );
     }
+
+    function partialLiquidationIsValid(
```

Comment:
Add documentation/NatSpec here

Reply:
Also, knowing which of these are WAD values would be helpful

Reply:
probably redundant since mock but done